### ISSUE TITLE
S3 batch delete gets 500

### ISSUE THREAD

**Issue by yaitskov** [localstack/localstack#2866]
Following POST request to S3 port returns 500.

http://localhost::4572/bucket1?delete=

request body:
```
<?xml version="1.0" encoding="UTF-8"?>
<Delete xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Object>
    <Key>folder/file1
    </Key>
  </Object>
  <Object>
    <Key>folder/file2
    </Key>
  </Object>
  <Object>
    <Key>folder/file3
    </Key>
  </Object>
</Delete>
```

response:
```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

docker logs is clean.

**Comment by yaitskov** [comment]
@whummer, could you point me where to look for a bug? 
I cannot trace to the code which is doing S3 logic.

**Comment by whummer** [comment]
Thanks for reporting @enzo-beltrami-ifood @yaitskov . This should be fixed in #2868 - can you please pull the `latest` image and give it another try? Please report here if the problem persists. Thanks

(Note: Generally speaking, you should be able to see some logs/stack traces if the `DEBUG=1` environment variable is configured in the LocalStack container. Please let us know if that is not the case. Thanks)

**Comment by yaitskov** [comment]
Thanks. It is fixed. Btw i tried to set DEBUG="1" in docker compose and didn't noticed any difference.
